GOVERNANCE.md

```diff
@@ -0,0 +1,192 @@
+- Responsive to issues and PRs assigned to them
+- Responsive to mentions of subprojects they are members of
+- Active owner of code they have contributed (unless ownership is explicitly transferred)
+  - Code is well-tested
+  - Tests consistently pass
+  - Addresses bugs or issues discovered after code is accepted
+- Members are welcome and encouraged to review PRs and proposals.
+- They can be assigned to issues and PRs, and people can ask members for reviews with a `/cc @username`.
+
+**Note:** Members who frequently contribute code are expected to proactively
+perform code reviews and work towards becoming a *core-contributor* for the
+subproject/repo that they are active in.
+
+### Core-Contributor
+
+Core-Contributors are able to both review and approve code contributions as well as 
+help maintainers triage issues and with project management.
+
+While code review is focused on code quality and correctness, approval is focused on
+holistic acceptance of a contribution including: backwards / forwards
+compatibility, adhering to API and flag conventions, subtle performance and
+correctness issues, interactions with other parts of the system, etc.
+
+**Defined by:** *approvers* entry in an OWNERS file in a repo owned by the
+Atlantis project.
+
+#### Requirements
+
+The following applies to the part of the codebase for which one would be an approver in an `OWNERS` file.
+
+- Member for at least 3 months
+- Author of at least 3 substantial PRs to the codebase, with the
+  definition of substantial subject to the lead's discretion (e.g.
+  refactors, enhancements rather than grammar correction or one-line pulls).
+- Exhibiting sound technical judgment through PR contributions
+- Exhibiting sound technical judgment through PR reviews
+- Sponsored by a Maintainer
+  - With no objections from other Maintainers or Steering Committee
+  - Done through PR to update the OWNERS file
+
+
+#### Responsibilities and privileges
+
+The following applies to the part of the codebase for which one would be a core-contributor
+in an `OWNERS` file.
+
+- Core-contributor status may be a precondition to accepting large code contributions
+- Demonstrate sound technical judgment
+- Responsible for project quality control via code reviews
+  - Focus on holistic acceptance of contributions such as dependencies with other features, backward / forwards
+    compatibility, API and flag definitions, etc
+- Expected to be responsive to review requests (inactivity may result in suspension until active again)
+- Mentor Members
+- May approve and merge code contributions for acceptance
+- Monitor Atlantis Slack (delayed response is perfectly acceptable), particularly for the area of your repository
+- Periodically attend the recurring community meetings
+- In general, continue to be willing to spend at least 5% of their time working on Atlantis (~0.25 business days per week)
```

Comment:
is this expected to be averages over a period of time or on a per week-to-week basis? If we want to be sensitive to responsiveness, sum'ing hours over 1 or 2 weeks makes sense, but if bursty contributions are fine, and latency is not important, looking at it monthly yields more flexibility for folks involved.